src/test/java/org/casbin/adapter/JdbcAdapterTest.java

```diff
@@ -41,7 +41,8 @@ public class JdbcAdapterTest {
      */
     @Test
     public void testLoadFilteredPolicy() {
-        JdbcAdapter.Filter filter = new JdbcAdapter.Filter();
+        org.casbin.jcasbin.persist.file_adapter.FilteredAdapter.Filter filter = new org.casbin.jcasbin.persist.file_adapter.FilteredAdapter.Filter();
```

Comment:
why use file-adapter?

Reply:
because it used  in JDBCAdapter：
```
filter instanceof Filter
```
this Filter use file-adapter.

Reply:
use ``import org.casbin.jcasbin.persist.file_adapter.FilteredAdapter.Filter``.

Reply:
fine